PythonTool/TagExpansion.py

```diff
@@ -114,20 +114,51 @@ def get_tag_lists(mb, element_type, element_id):
     return element_list, scalar_tags, vector_tags
 
 
-def create_database(mesh_file, mb_ref, mb_exp, elements, scal_tags, vec_tag, dir_name):
+def create_directory(mesh_file, main_dir_name = None):
```

Comment:
It appears that mesh_file is only passed here for the sake of naming the new files created directory, but there is also an optional main_dir_name argument. Could this be rolled into 1 single argument called output_dir? The directory created can have the same root name as the files generated, so there is no real reason they have to be two separate arguments.

If anything you could add an optional boolean argument for overwriting files already present. For example, True to overwrite a directory that already exists or False to make a new directory (appended w/ a number or so).